gulpfile.js

```diff
@@ -14,6 +14,57 @@ require('laravel-elixir-vue-2');
  */
 
 elixir(mix => {
-    mix.sass('app.scss')
-       .webpack('app.js');
+    mix
+        .styles(
+            [
+                'slick-carousel/slick/slick.css',
+                'slick-carousel/slick/slick-theme.css',
+                'eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
+            ],
+            'public/css/bower.css',
+            './resources/assets/bower_components/'
+        )
+        .styles(
+            [
+                '*.css',
```

Comment:
This captures all custom css?  You mentioned putting all our css files into one directory, does it have to be the one this gulpfile is in?  And would it be possible to do something like '**/*.css' in order to keep the css files all over?  (Not sure yet if I'll like them better all in one spot or with their respective pages, but in the end whatever works is good.)